### DISCUSSION TITLE
IAP file installation

### DISCUSSION THREAD

**Discussion by 1MM0R7AL5** [PlayCover/PlayCover#1731] [Q&A]
Would it be possible to install IAP (in-app purchase) files? I've made a few in-app purchases on some of my apps on my actual phone, and I want to dump those files into PlayCover.

**Comment by Catta1997** [comment]
Don't understand what you mean with "in-app purchase files" , if you are asking if you can buy something in-app or restore in-app purchase than no

**Reply by 1MM0R7AL5** [reply]
No, I mean dumping files that contain in-app purchases into PlayCover so you can reclaim them.

**Reply by Catta1997** [reply]
> No, I mean dumping files that contain in-app purchases into PlayCover so you can reclaim them.

like cheating? not supported and will never be

**Reply by 1MM0R7AL5** [reply]
No, no. Ripping ones you purchased from your phone and installing them on PlayCover.

**Reply by 1MM0R7AL5** [reply]
It's basically like sideloading, but with in-app purchases.
